site/content/api/Command-And-Control-API.md

```diff
@@ -16,50 +16,46 @@ The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e
 
 # Operations
 
-The following API defines operations that can be used by *Business Applications* to send control messages to devices over an *AMQP 1.0 Network* and how to get the response.
+The following API defines operations that can be used by *Business Applications* to send commands to devices over an AMQP 1.0 Network and get a response in return.
 
 ## Send a Command
 
 **Preconditions**
 
-1. The *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
-2. *Business Application* has established an AMQP link in role *sender* with the address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send control messages for and `${tenant_id}` is the ID of the tenant the device belongs to. 
+1. *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
+2. *Business Application* has established an AMQP link in role *sender* with the target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send messages to and `${tenant_id}` is the ID of the tenant that the device belongs to.
 
 **Link establishment and Message Flow**
 
-The following sequence diagram shows the whole message exchange, together with the reply to the *Business Application*.
+The following sequence diagram shows the whole message exchange including the response being sent back to the Business Application.
 
 ![Send Command](../command_control_Success.png)
 
 **Message Format**
 
-The following table provides an overview of the properties the *Business Application* needs to set on a *Control* message.
+The following table provides an overview of the properties the *Business Application* needs to set on a command message.
 
 | Name | Mandatory | Location | Type      | Description |
 | :--- | :-------: | :------- | :-------- | :---------- |
-| *message-id* | yes | *properties* | *string* | MUST contain the message ID used to match incoming control message with the response message containing result of execution. |
-| *subject* | yes | *properties* | *string* | MUST contain the command to be executed by a device. |
-| *content-type* | no | *properties* | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the sender should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
-| *ttl* | no | *properties* | *int* | Duration in milliseconds for which the message is to be considered 'live'. Messages that live longer than their expiration time will be discarded (or dead lettered). |
+| *message-id* | yes | *properties* | *string* | MUST contain the message ID used to match incoming command message with the response message containing the result of execution. |
```

Comment:
do we also allow to specify a separate *correlation-id* like we do in the other request/response APIs?

Reply:
I agree, that we should handle this consistent. Why do we allow this in the other API's? Is there a need for an own ID?

Reply:
IIRC we wanted to make sure that a client can exlicitly set a correlation ID on an outgoing message in order to prevent a race condition when matching the response. This problem could occur when sending a request using JMS where the message ID cannot be set by the client code but is set by the JMS provider and thus is only available *after* the message has been sent ...

Reply:
I see, that makes sense - so I will add it to Command and Control.